views/patterns/02-organisms/global/footer.blade.php

```diff
@@ -67,7 +67,6 @@
   </div> <!-- /.c-footer--inner -->
 </footer> <!-- /.c-footer -->
 
-<script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/2.2.4/jquery.min.js"></script>
```

Comment:
Can this be blocked out **_IF_** Wordpress is version `5.6` or newer? Could something like `is_wp_version_compatible()` or `wp_version_check()`? 

I don't want to break this for older versions, if we can help it.

Reply:
We discused